python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/vmtracer-sessions.j2

```diff
@@ -10,15 +10,15 @@
 
 #### VM Tracer Summary
 
-| Session | URL | Username | Autovlan | Source Interface |
-| ------- | --- | -------- | -------- | ---------------- |
+| Session | URL | Username | Autovlan | VRF | Source Interface |
+| ------- | --- | -------- | -------- | --- | ---------------- |
 {%     for session in vmtracer_sessions | arista.avd.natural_sort('name') %}
 {%         set url = session.url | arista.avd.default('-') %}
 {%         if session.autovlan_disable is arista.avd.defined(true) %}
 {%             set autovlan = 'disabled' %}
 {%         endif %}
 {%         set source_interface = session.source_interface | arista.avd.default('-') %}
-| {{ session.name }} | {{ url }} | {{ session.username | arista.avd.default('-') }} | {{ autovlan | arista.avd.default('enabled') }} | {{ source_interface }} |
+| {{ session.name }} | {{ url }} | {{ session.username | arista.avd.default('-') }} | {{ autovlan | arista.avd.default('enabled') }} | {{ session.vrf | arista.avd.default('-') }} | {{ source_interface }} |
```

Comment:
here we are mixing a bit the set entries and the inline ones which can be a bit tricky to follow I suggest we put VRF as source_interface above so that it is easy to review